base/libc.jl

```diff
@@ -335,45 +338,6 @@ if Sys.iswindows()
     end
 end
 
-## Memory related ##
```

Comment:
These don't appear to be necessary to move. They therefore should stay here.

Reply:
We don't appear to need any of the malloc/free/etc you removed from here, just the memcpy/memmove/memset/memcmp you added

Reply:
Ah, I see. I was just trying to keep the 'mem' stuff together, but I can put the other ones back